pkg/integrations/manager.go

```diff
@@ -68,6 +71,19 @@ type ManagerConfig struct {
 	// ListenHost tells the integration Manager which port the Agent is
 	// listening on for generating Prometheus instance configs
 	ListenHost *string `yaml:"-"`
+
+	// Drives using HTTPS to read the integrations
+
+	ClientCert string `yaml:"client_cert_file"`
+	ClientKey  string `yaml:"client_key_file"`
+	ServerName string `yaml:"server_name"`
+
+	// If this is RequireAndVerifyClientCert then the ClientCert/Key/ServerName fields are required
+	// comes from the http_tls_config in the server settings
+	ClientAuthType string `yaml:"-"`
+
+	// Signing authority, needed if self signed or otherwise untrusted
+	ClientCA string `yaml:"-"`
```

Comment:
From offline discussion: we can just add the Prometheus' TLS client settings directly here.

Reply:
Done!